test/pylib: allow run minio_server.py as a stand-alone tool

tchaikov opened the pull request:
this would allow developer to run a minio server for testing, for instance, s3_test.

tchaikov commented:
@xemul hi Pavel, could you help review this change as well?

scylladb-promoter commented:
CI state **`SUCCESS`** - https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/2196/

xemul reviewed (changes requested):
Please, add the shell command example how to launch the thing standalone. E.g. in patch comment. Otherwise it's awesome

tchaikov commented:
v2
- revise the commit message to demonstrate how the script can be used as a stand-alone tool.
- add another commit to make the script more user friendly

@xemul hi Pavel, thank you for your review. i think i've addressed your comment. could you please take another look?

scylladb-promoter commented:
CI state **`SUCCESS`** - https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/2222/
